app/addons/documents/tests/changes.componentsSpec.react.jsx

```diff
@@ -0,0 +1,215 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+  'react',
+  'addons/documents/changes/components.react',
+  'addons/documents/changes/stores',
+  'addons/documents/changes/actions',
+  'testUtils'
+], function (app, FauxtonAPI, React, Changes, Stores, Actions, utils) {
+  FauxtonAPI.router = new FauxtonAPI.Router([]);
+
+  var assert = utils.assert;
+  var TestUtils = React.addons.TestUtils;
+
+
+  describe('ChangesHeader', function () {
+    var container, tab, spy;
+
+    describe('Testing DOM', function () {
+      beforeEach(function () {
+        spy = sinon.spy(Actions, 'toggleTabVisibility');
+        container = document.createElement('div');
+        tab = TestUtils.renderIntoDocument(<Changes.ChangesHeader />, container);
+      });
+
+      afterEach(function () {
+        Stores.changesFilterStore.reset();
+        React.unmountComponentAtNode(container);
+      });
+
+      // similar as previous, except it confirms that the action gets fired, not the custom toggle func
+      it('calls toggleTabVisibility action on selecting a tab', function () {
+        TestUtils.Simulate.click($(tab.getDOMNode()).find('a')[0]);
+        assert.ok(spy.calledOnce);
+      });
+    });
+
+    it('toggleTabVisibility() changes state in store', function() {
```

Comment:
Can you move the store tests into their own stores test file. It doesn't make sense to have them in the components test file.